src/core/electric_load.jl

```diff
@@ -109,7 +109,7 @@ mutable struct ElectricLoad  # mutable to adjust (critical_)loads_kw based off o
         off_grid_flag::Bool = false,
         loads_kw::Array{<:Real,1} = Real[],
         path_to_csv::String = "",
-        year::Int = 2020, # used in ElectricTariff to align rate schedule with weekdays/weekends
+        year::Int = 2020, # used in ElectricTariff to align rate schedule with weekdays/weekends # TODO: why don't we default this to 2017 to avoid warning each time?
```

Comment:
@Bill-Becker Do you happen to know why we default the ElectricLoad year to 2020, but then always change it to 2017 and warn the user? Why not just default to 2017?

Reply:
@Bill-Becker I ended up implementing a conditional default ElectricLoad "year" value based on whether or not the user is providing their own data or using a CRB. I think it makes sense to assume by default that user-provided data would be for the latest year available (2022). What do you think of this change? https://github.com/NREL/REopt.jl/pull/181/commits/a36195027e7b5a31978e19b00c357644e2f50e3d